dashboard/app/models/block.rb

```diff
@@ -35,6 +39,18 @@ def write_block_files
     File.write js_path, helper_code
   end
 
+  def delete_block_files(old_level_type=level_type, old_name=name)
```

Comment:
Can we add a test to make sure the files are deleted (or is it already covered by an existing test)?